Thanks for the PR!
Looks like a great feature, unfortunately I get this error when trying to build

```sh
✘ [ERROR] Could not resolve "./onCC"

    src/hydra-api/index.ts:5:14:
      5 │ export * from "./onCC";
        ╵               ~~~~~~~~

IIFE Build failed
Error: Build failed with 1 error:
src/hydra-api/index.ts:5:14: ERROR: Could not resolve "./onCC"
```

There is no file `onCC` in `src/hydra-api/`

--- a/src/hydra-api/onCC.ts
+++ b/src/hydra-api/onCC.ts
@@ -1,0 +1,15 @@
+import state from '../state'
+import { ChannelArg, InputArg, IndexArg, CcEventContext } from '../types'
+import { getCcId } from './cc'
+
+export const onCC = (
+  index: IndexArg,
+  channel: ChannelArg,
+  input: InputArg,
+  event: (context: CcEventContext) => void
+) => {
+  const ccId = getCcId(index, channel, input)
+  state.CcEvents.set(ccId, event)
+}
+
+

